flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -47,8 +47,8 @@
 import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
 import org.apache.flink.table.api.internal.TableEnvironmentInternal;
 import org.apache.flink.testutils.junit.FailsOnJava11;
-import org.apache.flink.testutils.junit.RetryOnException;
 import org.apache.flink.testutils.junit.RetryRule;
+import org.apache.flink.testutils.junit.extensions.retry.annotation.RetryOnException;
```

Comment:
I'm wondering if we should revert this change. It can just easily lead to issues during rebases/backports.

Reply:
These annotations are used by Junit4 and Junit5 tests before we totally migrate to Junit5. Should I put them back to the package they used to be?

Reply:
yes; that's what I meant.